main.go

```diff
@@ -377,17 +380,47 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// Get Uniswap data
-	uniswapPoolData := getPoolPairData(client, common.HexToAddress(config["poolPairAddress"]))
+	var wg sync.WaitGroup
+	var uniswapPoolData *models.ProtocolData
+	var addresses []*models.LMCAddress
+	var campaigns []*models.LMCampaign
+	var positions []*models.UserPosition
+
+	wg.Add(2)
+
+	go func() {
+		// Get Uniswap data
+		uniswapPoolData = getPoolPairData(client, common.HexToAddress(config["poolPairAddress"]))
+
+		wg.Done()
```

Comment:
When go routines get bigger, it can happen that you forget to all Done(). Get into the habbit to call it as `defer wg.Done()`  as the first line in the go routine. This way it always gets called (even if the go routine panics) and you don't forget to call it at the end.

Reply:
Done